src/citrine/resources/status_detail.py

```diff
@@ -30,3 +30,6 @@ def __init__(self, *, msg: str, level: Union[str, StatusLevelEnum]):
 
     def __str__(self):
         return f"[{self.level.upper()}] {self.msg}"  # pragma: no cover
+
+    def __repr__(self):
+        return f"<{str(self)}>"  # pragma: no cover
```

Comment:
Why no cover?

Reply:
Laziness. But you're right, I'll go add a test for both `__str__` and `__repr__`.